### PR TITLE
Update sentiment-analysis.md

### PR DESCRIPTION
**Pull request by cassws**
Added Acknowledgements section (neglected to copy it over from Part 2!)

### CODE REVIEW

File: lessons/sentiment-analysis.md [hunk 1]
```diff
@@ -397,6 +397,12 @@ But even at this level, *VADER* also runs into a number of errors. The sentence
 
 What do you notice about the distribution of scores? How can you imagine collecting them in a manner that would help you better understand your data and its relationships to the research questions you care about? (Feel free to experiment with different kinds of text in the *message_text* variable to see how the tool responds to different types of language constructions). The code you have just written can be repurposed for any text.
 
+# Acknowledgments
+
+My sincere thanks to Justin Joque, Visualization Librarian at the University of Michigan Library and the Digital Project Studio for support in formulating the ideas and approach behind this lesson. More information about the Digital Project Studio can be found at https://clarkdatalabs.github.io
```
Comment (ianmilligan1) [inline lessons/sentiment-analysis.md]: Can you make https://clarkdatalabs.github.io either a clickable link (wrap in `<` and `>`) or maybe even more simply just make the first mention of "Digital Project Studio" a link to the site?

**Review by ianmilligan1 — CHANGES_REQUESTED** [review]

**Review by ianmilligan1 — APPROVED** [review]

**Comment by ianmilligan1** [comment]
As @acrymble is editor I'll give him a chance to review and/or merge. Thanks @zoews!

**Comment by acrymble** [comment]
@zoews can you update the "last modified date" in the header please?

**Comment by mdlincoln** [comment]
@acrymble That bit of the header is not part of the YAML metadata, but is instead generated by an API call to github. Unfortunately, it seems to be behaving strangely - right now I see `2018-00-22` for this lesson, which is obviously nonsensical. I'm going to merge this first, and then open a new issue about the modified date here because it points to a technical problem that may go beyond this lesson.

**Pull request merged**